src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_TabKey.cs

```diff
@@ -82,7 +83,7 @@ private bool TryInvokeSnippetCompletion(TabKeyCommandArgs args, ICompletionServi
                         {
                             var textChange = new TextChange(TextSpan.FromBounds(caretPoint - 1, caretPoint), string.Empty);
                             workspace.ApplyTextChanges(documentId, textChange, CancellationToken.None);
-                            this.StartNewModelComputation(completionService, CompletionTriggerInfo.CreateSnippetTriggerInfo(), filterItems: false);
+                            this.StartNewModelComputation(completionService, DisplaySnippetsCompletionTrigger.Instance, filterItems: false);
```

Comment:
Here you have a static instance.  Earlier you were instantiating.  I'd like this to be consistent.